src/platform/plugins/shared/dashboard/public/dashboard_api/types.ts

```diff
@@ -157,6 +158,10 @@ export type DashboardApi = CanExpandPanels &
     unifiedSearchFilters$: PublishesUnifiedSearch['filters$'];
   };
 
+export interface SerializeControlsResult {
+  controlGroupInput: ControlsGroupState | undefined;
+  controlGroupReferences: DashboardState['references'];
```

Comment:
Maybe let's just use the `Reference` type we've already imported.
 
```suggestion
  controlGroupReferences: Reference[];
```